src/cmd/lightning/check.rs

```diff
@@ -0,0 +1,26 @@
+use crate::cmd::{Cmd, LightningTopUpInfo};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CheckLightningTopUpInfo {
+    status: LightningTopUpInfo,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub enum LightningTopUpStatus {
+    Unpaid,
+    Paid,
+    Expired,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CheckLightningTopUp {
+    pub id: String,
+}
+
+impl Cmd for CheckLightningTopUp {
+    type Output = CheckLightningTopUpInfo;
+
+    const METHOD: http::Method = http::Method::POST;
+    const PATH: &'static str = "lightning/top_up/check";
```

Comment:
This isn't really nice, it should be a GET. But we currently log URLs so don't really have a good way to pass the parameter. Probably we should just disable the URL info from the NGINX access logs and use GET lightning/top_up/{id}

But whatever, this is fine.